### PR TITLE
fixes for treshold keys and reuse old privateKeys

### PR DESCRIPTION
**Pull request by Coiling-Dragon**
**Detailed description**:

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: scripts/bridge/setup/from-config/cmd/setup-from-config.go [hunk 1]
```diff
@@ -50,20 +50,35 @@ func main() {
...
 	configPath := flag.String("configPath", "scripts/bridge/setup/extend-config/extended-bridge.yml", "Path to the 'bridge.yaml' config file")
 	flag.Parse()
 
+	prKeysSlice := strings.Split(*memberPrivateKeys, ",")
+	var hederaPrivateKeys []hedera.PrivateKey
+	for i := 0; i < len(prKeysSlice); i++ {
+		// element [0] can be empty string if the user does not provide any private keys
+		if prKeysSlice[0] == "" {
+			break
+		}
```
Comment (rokn) [inline scripts/bridge/setup/from-config/cmd/setup-from-config.go:66]: I think this requires some mind gymnastics to understand, so we should either:
1. Surround the for with the if instead
2. Extract this logic in a function which returns the private keys array from the string and do:
```golang
if prKeysSlice[0] == "" {
	return nil
}
```

File: scripts/bridge/setup/setup.go [hunk 2]
```diff
@@ -37,9 +38,11 @@ type DeployResult struct {
 	Error              error
 }
 
-func Deploy(privateKey *string, accountID *string, adminKey *string, network *string, members *int, topicThreshold *uint) DeployResult {
+func Deploy(privateKey *string, accountID *string, adminKey *string, network *string, members *int, cachedPrivateKeys []hedera.PrivateKey, topicThreshold *uint) DeployResult {
```
Comment (rokn) [inline scripts/bridge/setup/setup.go:41]: I think `oldPrivateKeys` or `previousPrivateKeys` conveys a bit more clearer what those keys are

File: scripts/bridge/setup/setup.go [hunk 3]
```diff
@@ -37,9 +38,11 @@ type DeployResult struct {
...
 	result := DeployResult{}
 
+	fmt.Printf("%v", cachedPrivateKeys)
```
Comment (rokn) [inline scripts/bridge/setup/setup.go:44]: nit: If it's not leftover add some more info

File: scripts/bridge/setup/cmd/setup.go [hunk 4]
```diff
@@ -22,7 +22,6 @@ import (
 	"github.com/limechain/hedera-eth-bridge-validator/scripts/bridge/setup"
 )
 
-var balance = hedera.NewHbar(100)
```
Comment (rokn) [inline scripts/bridge/setup/cmd/setup.go:25]: Wasn't this used somewhere?

Comment (Coiling-Dragon) [reply scripts/bridge/setup/cmd/setup.go:25]: unused variable

File: scripts/bridge/setup/setup.go (hunk 3 above)
Comment (Coiling-Dragon) [reply scripts/bridge/setup/setup.go:44]: forgot to delete it

File: scripts/bridge/setup/from-config/cmd/setup-from-config.go (hunk 1 above)
Comment (Coiling-Dragon) [reply scripts/bridge/setup/from-config/cmd/setup-from-config.go:66]: Noted and refactored to:
```
	// element [0] can be empty string if the user does not provide any private keys
	if prKeysSlice[0] != "" {
		for i := 0; i < len(prKeysSlice); i++ {
			privateKeyFromStr, err := hedera.PrivateKeyFromString(prKeysSlice[i])
			if err != nil {
				panic(err)
			}
			hederaPrivateKeys = append(hederaPrivateKeys, privateKeyFromStr)
		}
	}
```

**Review by rokn — APPROVED** [review]

**Pull request merged**